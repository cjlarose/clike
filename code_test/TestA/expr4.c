



void printint(int);

void main()
{
  char c;
  int x;
  int y;

  c = -120;  // Does sign extension work?
  x = 100;

  y = c + x;
  printint(y) ; // Precedence work?
}
  
