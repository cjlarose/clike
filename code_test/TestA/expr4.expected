Loaded: /usr/local/share/spim/exceptions.s
-20